src/arrayMethodReduce.js

```diff
@@ -4,8 +4,19 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function(callback, ...initialValue) {
+    const prev = initialValue;
+
+    for (let i = 0; i < this.length; i++) {
```

Comment:
create a startIndex variable with default value 0? and use it instead i = 0, because there may be a case when only the callback will be given (in this case accumulator value = this[0], startIndex = 1)

Reply:
👌 fixed